doc/sphinx-guides/source/admin/user-administration.rst

```diff
@@ -56,3 +56,14 @@ Using the API token 7ae33670-be21-491d-a244-008149856437 as an example:
 
 You should expect the output ``DELETE 1`` after issuing the command above.
 
+Combining Accounts
+------------------
+
+If a user has created multiple accounts and has been performed actions under both accounts that need to be preserved, these accounts can be combined.  One account can be merged into another account and all data associated with both accounts will be combined in the surviving account.
+
+This command::
+
+	curl -X PUT -d http://$server/api/admin/authenticatedUsers/$mergedIdentifier/mergeIntoUser/$continuingIdentifier
+
+moves account data from $mergedIdentifier into the account $continuingIdentifier and deletes the account of $mergedIdentifier.
```

Comment:
I like how descriptive "continuing identifier" is but I would suggest that "to merge identifier" might make a little more sense that "merged identifier" to someone who is preparing to perform the merge by reading this guide.